client/verta/verta/tracking/entities/_experimentrun.py

```diff
@@ -132,8 +132,8 @@ def workspace(self):
             # workspace is OSS default
             return self._conn._OSS_DEFAULT_WORKSPACE
         else:
-            return self._conn.get_workspace_name_from_legacy_id(
-                project_json["workspace_id"]
+            return self._conn.get_workspace_name_from_id(
+                project_json["workspace_service_id"]
```

Comment:
Sorry this is an extra request, but could we do this so it is more similar to the other `def workspace()`s?

```python
    @property
    def workspace(self):
        self._refresh_cache()

        msg = ProjectService_pb2.GetProjectById(id=self._msg.project_id)
        url = "/api/v1/modeldb/project/getProjectById"
        response = self._conn.make_proto_request("GET", url, params=msg)
        response = self._conn.must_proto_response(response, msg.Response)
        proj_proto = response.project

        if proj_proto.workspace_service_id:
            return self._conn.get_workspace_name_from_id(proj_proto.workspace_service_id)
        else:
            return self._conn._OSS_DEFAULT_WORKSPACE
```

(will require a `from verta._protos.public.modeldb import ProjectService_pb2` at the top of the file)

Reply:
applied your code snippet